packages/catalog-model/examples/acme/infrastructure-group.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: backstage.io/v1alpha1
+kind: Group
+metadata:
+  name: infrastructure
+  description: The infra department
+spec:
+  type: department
+  parent: acme-corp
+  ancestors: [acme-corp]
+  children: [backstage, boxoffice]
+  descendants: [backstage, boxoffice, team-a, team-b, team-c, team-d]
```

Comment:
This is very not-DRY. If team-b disappears from the hierarchy, now I have to fix multiple files.
Why can't we just do parent and children and create the tree from that?

Reply:
Yep! These fields are about to disappear, as part of building out the [concept of relations](https://github.com/spotify/backstage/issues/1964). Work is undergoing at this very moment on that. You would for example in the end also want to only express `Group.spec.parent`, or only `Group.spec.children` (depending on what direction your data source expresses them as), not both - they form the same tree either way